flink-table/flink-table-planner/pom.xml

```diff
@@ -353,19 +353,6 @@ under the License.
 							<pattern>org.checkerframework</pattern>
 							<shadedPattern>org.apache.flink.calcite.shaded.org.checkerframework</shadedPattern>
 						</relocation>
-
-						<!-- These are relocated to match package names, but not included in this jar -->
-						<relocation>
-							<pattern>com.fasterxml</pattern>
-							<!-- From flink-shaded-jackson -->
-							<shadedPattern>org.apache.flink.shaded.jackson2.com.fasterxml</shadedPattern>
-						</relocation>
-						<relocation>
-							<pattern>com.jayway</pattern>
-							<!-- From flink-table-runtime -->
-							<shadedPattern>org.apache.flink.table.shaded.com.jayway</shadedPattern>
-						</relocation>
-
```

Comment:
We use a shaded `jsonpath` version, which has those relocations applied already.